### PR TITLE
Add and upgrade to Milvus2.0 support for neural search

### PR DESCRIPTION
**Pull request by w5688414**
### PR types

+ New features

### PR changes

+ APIs

### Description

+ upgrade to Milvus2.0 support for neural search

### CODE REVIEW

**Review by tianxin1860 — DISMISSED** [review]
LGTM

File: applications/neural_search/recall/in_batch_negative/export_model.py [hunk 1]
```diff
@@ -34,22 +34,20 @@
 # yapf: enable
 
 if __name__ == "__main__":
-    # If you want to use ernie1.0 model, plesace uncomment the following code
     output_emb_size = 256
-
-    pretrained_model = AutoModel.from_pretrained("ernie-3.0-medium-zh")
-
-    tokenizer = AutoTokenizer.from_pretrained('ernie-3.0-medium-zh')
```
Comment (tianxin1860) [inline applications/neural_search/recall/in_batch_negative/export_model.py:42]: 不用回退到 ERNIE1.0, 默认推广 ERNIE3.0-Medium 就好。

**Review by tianxin1860 — COMMENTED** [review]
Leave a comment

File: applications/neural_search/recall/in_batch_negative/export_model.py (hunk 1 above)
Comment (w5688414) [reply applications/neural_search/recall/in_batch_negative/export_model.py:42]: 默认的预训练模型是基于ernie1.0训练的，如果代码切换成ernie3.0，导出会有点问题

**Review by tianxin1860 — APPROVED** [review]
LGTM

**Pull request merged**